app/pods/publications/publication/proofs/index/template.hbs

```diff
@@ -0,0 +1,74 @@
+<div class="auk-u-mt-4">
+  <Auk::Panel>
+    <Auk::Panel::Header>
+      <Auk::Toolbar>
+        <Auk::Toolbar::Group @position="left">
+          <Auk::Toolbar::Item>
+            <h4 class="auk-panel__title">
+              {{t "proofs"}}
+            </h4>
+          </Auk::Toolbar::Item>
+        </Auk::Toolbar::Group>
+        <Auk::Toolbar::Group @position="right">
+          <Auk::Toolbar::Item>
+            <Auk::Button
+              @skin="secondary"
+              @icon="plus"
+              @disabled={{this.isProofUploadDisabled}}
+              {{on "click" this.openProofUploadModal}}
+            >
+              {{t "upload-proof"}}
+            </Auk::Button>
+          </Auk::Toolbar::Item>
+          <Auk::Toolbar::Item>
+            <Auk::Button
+              @skin="primary"
+              @icon="plus"
+              {{on "click" this.openProofRequestModal}}
+            >
+              {{t "request-proof"}}
+            </Auk::Button>
+          </Auk::Toolbar::Item>
+        </Auk::Toolbar::Group>
+      </Auk::Toolbar>
+    </Auk::Panel::Header>
+    <Auk::Panel::Body class="auk-u-bg-alt">
+      {{#each @model as |timelineActivity|}}
+        {{#if timelineActivity.isRequestActivity}}
+          <Publications::Publication::Proofs::ProofRequestedPanel
+            @requestActivity={{timelineActivity.activity}}
+            @onDeleteRequest={{fn
+              (perform this.deleteRequest)
+              timelineActivity.activity
+            }}
+          />
+        {{else if (and
+          timelineActivity.isProofingActivity timelineActivity.activity.endDate
+        )}}
+          <Publications::Publication::Proofs::ProofReceivedPanel
+            @proofingActivity={{timelineActivity.activity}}
+            @publicationSubcase={{this.publicationSubcase}}
```

Comment:
De hele publication-subcase wordt meegegeven om dan enkel de `proofprintCorrector` te gebruiken als initialisatie waarde. Kunnen we die apart doorgeven (zoals bijvoorbeeld ook `@dueDate` bij vertalingen)